src/libs/ReportUtils.js

```diff
@@ -860,12 +860,7 @@ function getReportName(report, policies = {}) {
     const participantsWithoutCurrentUser = _.without(participants, sessionEmail);
     const isMultipleParticipantReport = participantsWithoutCurrentUser.length > 1;
 
-    const displayNames = [];
-    for (let i = 0; i < participantsWithoutCurrentUser.length; i++) {
-        const login = participantsWithoutCurrentUser[i];
-        displayNames.push(getDisplayNameForParticipant(login, isMultipleParticipantReport));
-    }
-    return displayNames.join(', ');
+    return _.map(participantsWithoutCurrentUser, login => getDisplayNameForParticipant(login, isMultipleParticipantReport)).join(', ');
```

Comment:
@tgolen Ended up improving this block with your suggestion as well, which it's funny because you wrote it haha

Reply:
hahaha, future Tim is much smarter than past Tim!

I think what probably happened is I moved this code from somewhere else and didn't want to change too much logic when I did a big refactor.